specification/cognitiveservices/Language.QuestionAnswering-authoring/examples/2025-05-15-preview/questionanswering/authoring/SuccessfulImportProjectFromFile.json

```diff
@@ -0,0 +1,22 @@
+{
+  "operationId": "QuestionAnsweringProjects_ImportFromFiles",
+  "title": "Create project from file",
+  "parameters": {
+    "Content-Type": "multipart/form-data",
+    "Endpoint": "{Endpoint}",
+    "Ocp-Apim-Subscription-Key": "{API key}",
+    "api-version": "2025-05-15-preview",
+    "files": [
+      "{binary file}"
+    ],
+    "projectName": "proj1"
+  },
+  "responses": {
+    "202": {
+      "description": "A successful call results with an Operation-Location header used to check the status of the analysis job.",
```

Comment:
nit: grammatically "results with" is generally not used; this looks like it meant to be "results in"?  (Update: this is copy/pasted in various other places)